active/package-maintainers.md

```diff
@@ -0,0 +1,126 @@
+
+## Reporting
+
+The group will share highlights from current activities on a monthly basis via a forum post.
+
+## Appendix
+
+### Group activities
+
+As an illustration of the group’s remit, here are possible activities members could take part in.
+
+#### Support Django version compatibility efforts
+
+As Django regularly releases new versions, it’s an ongoing effort to ensure packages are compatible with new releases.
+
+Group members can support maintainers in this effort by:
+
+- Creating a shared calendar with Django release dates and deadlines.
+- Creating communication channels for package maintainers to coordinate compatibility changes for a specific release.
+- Recommending appropriate automation, such as [django-upgrade](https://github.com/adamchainz/django-upgrade) or how to set up Django pre-releases in Continuous Integration.
+- Proposing additions to the Django release notes with recommendations on how to handle deprecations.
+
+#### Share packaging best practices
+
+The group can work on create, maintain, or curate best practices for Django package maintainers. For example, the official [Advanced tutorial: How to write reusable apps](https://docs.djangoproject.com/en/5.0/intro/reusable-apps/), or packaging-focused sections on third-party resources like [awesome-django](https://github.com/wsvincent/awesome-django).
+
+Those best practices can cover aspects like:
+
+- Documentation approaches and tools
+- Use of the `django_` prefix for app names (see [Change reusable apps naming recommendation](https://forum.djangoproject.com/t/change-reusable-apps-naming-recommendation/25233)).
+- Recommended Python, Django, browser, operating system support policies to align with those of Django itself.
+- Tooling, for example how to set up Continuous Integration for Django packages.
+
+#### Facilitate community-driven maintenance
+
+Group members can liaise with organizations dedicated to sharing maintenance efforts, such as [Jazzband](https://jazzband.co/), [Wagtail Nest](https://github.com/wagtail-nest), or [Django Commons](https://github.com/django-commons). Or work directly with packages in need of new maintainers to find new candidates. This can involve:
+
+- Sharing calls for new package maintainers
+- Coordinating trusted volunteer open source “roadies” familiar with package ownership transfers.
+- Facilitating participation in mentoring programs such as [Google Summer of Code](https://summerofcode.withgoogle.com/) or [Djangonaut Space](https://djangonaut.space/).
+- Providing support for tasks requiring very specific expertise, like funding avenues or vulnerability reports handling.
+
+#### Review the package ecosystem
+
+The group can organize periodic reviews of the package ecosystem to assess its health, for other efforts to make more informed decisions.
+
+- Statistics on compatible Django / Python versions, or use of type annotations
+- Package health and popularity metrics
```

Comment:
+1 for package health, -1 for popularity metrics. Having PyPI download counts could be a neutral middle way of sharing "people use it" (which still doesn't directly correlate to value), but anything else like Github stars or contributor numbers would harm the packages that doesn't need to increse those numbers to be beneficial.